### PR TITLE
feat: Docker-compose and Hydra config

### PR DESCRIPTION
**Pull request by Nesqwik**
## Description of the changes
Setup the Hydra Dockerfile and config for local development. 

## Checklist
- [x] I didn't over-scope my PR
- [x] My PR title matches the [commit convention](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I did not include breaking changes
- [x] I made my own code-review before requesting one

### I included unit tests that cover my changes
  - [ ] 👍 yes
  - [x] 🙅 no, because they aren't needed
  - [ ] 🙋 no, because I need help

### I added/updated the documentation about my changes

- [x] 📜 README.md
- [ ] 📕 docs/*.md
- [ ] 📓 docs.lenra.io
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,11 @@ You will then need to install and setup elixir prerequisites for the server to r
   * `mix ecto.migrate` to start all migration and have an up-to-date database
   * `mix run priv/repo/seeds.exs` to fill database with default values
 
+Finally, start Ory Hydra using the docker-compose.yml file : 
+```
+docker-compose up -d
```
Comment (pichoemr) [inline README.md:74]: Maybe update the previous docker-compose command in the README to include the name docker-compose.databse.
(so as not to confuse it)

File: README.md [hunk 2]
```diff
@@ -54,11 +54,15 @@
 You will first need to start two databases, postgres and mongo. Postgres will be used by the server to store general data and Mongo will store the data of the applications that you run.
 You can do this by using the 
 
-`docker compose up` 
+`docker compose up -d`
```
Comment (taorepoara) [inline README.md:57]: The new lines are not needed since it's not a list anymore

Comment (Nesqwik) [reply README.md:57]: It wasn't a list before eather :p 
Fixed

Comment (taorepoara) [reply README.md]: My bad, it was the `-` of the diff ^^

**Pull request merged**